client/network/src/protocol.rs

```diff
@@ -784,7 +793,7 @@ impl<B: BlockT, H: ExHashT> Protocol<B, H> {
 		request: message::BlockRequest<B>,
 		response: message::BlockResponse<B>,
 	) -> CustomMessageOutcome<B> {
-		let blocks_range = match (
+		let blocks_range = || match (
```

Comment:
Is this change required?

Reply:
No, this is an optimization not to format strings when tracing is disabled.

Reply:
Ahh, didn't github folded the `trace!` call. Makes sense :)